README.md

```diff
@@ -20,28 +20,28 @@ Below, we illustrate a minimal sample configuration with the international SQLit
 
 > warning: This beta version  of the Launcher 2.0 is for test purpose only and should be used with our German sandbox.
 
-Download the latest release from GitHub.
+Download the latest release from GitHub. We always recommend using the latest release to benefits from the newest improvements.
 Unzip the downloaded release.
 
-### Supported Packages in the Alpha
+### <a name="supported-packages-in-the-alpha"></a>Supported Packages in the Alpha
 
 In the sandbox configure a cashbox based on the list of supported packages below.
 
 | Name                                           | Versions    |
 | ---------------------------------------------- | ----------- |
 | fiskaltrust.Middleware.Queue.MySQL             | v1.3.37     |
-| fiskaltrust.Middleware.Queue.SQLite            | v1.3.38-ci-22221-55477     |
+| fiskaltrust.Middleware.Queue.SQLite            | v1.3.38-rc1-22249-56008 8-ci-22221-55477     |
 | fiskaltrust.Middleware.SCU.DE.FiskalyCertified | v1.3.35     |
 | fiskaltrust.Middleware.SCU.DE.CryptoVision     | v1.3.34 |
 | fiskaltrust.Middleware.SCU.DE.DeutscheFiskal   | v1.3.35     |
 | fiskaltrust.Middleware.SCU.DE.DieboldNixdorf   | v1.3.20     |
 | fiskaltrust.Middleware.SCU.DE.Epson            | v1.3.19     |
 | fiskaltrust.Middleware.SCU.DE.Swissbit         | v1.3.35-rc1    |
 | fiskaltrust.Middleware.SCU.DE.SwissbitCloud    | v1.3.35     |
-| fiskaltrust.Middleware.Helper.Helipad          | v1.3.26     |
+| fiskaltrust.Middleware.Helper.Helipad          | v1.3.33     |
 
 
-### Launcher configuration
+### <a name="launcher-configuration"></a> Launcher configuration
```

Comment:
```suggestion
### Launcher configuration
```
 same as above.

Reply:
Sorry I added that because when i tested in Visual studio code the links created in the guide did not work when using the heading